src/main/java/movieDB/Movie.java

```diff
@@ -0,0 +1,126 @@
+package movieDB;
+
+import java.sql.*;
+
+/**
+ * Created by melmo on 12/13/16.
+ */
+public class Movie {
+    private int movieId = 0;
+    private String title;
+    private String genres;
+    private String mpaaRating;
+    private int year;
+    private int avgRating;
+
+    public static  Movie getMovie(int id) throws SQLException {
+        Connection connection = DriverManager.getConnection("jdbc:postgresql:moviescopy");
+        Statement statement = connection.createStatement();
+        ResultSet result = statement.executeQuery(
+                "SELECT * FROM movies WHERE movieid = " + id);
+        result.next();
+        int movieId = result.getInt("movieid");
+        String title = result.getString("title");
+        String genres = result.getString("genres");
+        String mpaaRating = result.getString("mpaa_rating");
+        int year = result.getInt("year");
+        int avgRating = result.getInt("avg_rating");
+
+        return new Movie(movieId, title, genres, mpaaRating, year, avgRating);
+    }
+
+    public void save() throws SQLException {
```

Comment:
This is close.  You should always update the `id` field from the database on insert.  If you don't then 2 problems arise. 1. You aren't sure the database did what you asked.  2. The user has to do another query to the database to refresh the object which is a waste if you are already doing the work.  So in the future add the call to get the generated keys.